torch/fx/graph.py

```diff
@@ -1625,6 +1625,34 @@ def forward(self, x):
 
         return changed
 
+    def _eliminate_dead_code_custom(self, is_impure_node: Callable[[Node], bool]):
```

Comment:
I think this can just be a signature change to `eliminate_dead_code()`?

Reply:
fixed now!

Reply:
nice! this looks great